### ISSUE TITLE
Question about arrow buttons and horizontal mode

### ISSUE THREAD

**Issue by Porkepix** [iamrasel/lunar-launcher#122]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/iamrasel/lunar-launcher/issues) or [closed](https://github.com/iamrasel/lunar-launcher/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have already checked the [WIKI](https://github.com/iamrasel/lunar-launcher/wiki) pages.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.

### What is/are your question(s)?

In the app drawer screen, the small letter list at the right of the screen does absolutely nothing. I feel like it's supposed to be a quick access to the application list at these letters, but can't be sure.
Also, the expected effect (for me) of the arrow buttons under it would be to move a little bit lower or higher, similarly to the scrolling. Instead it moves completely at the beginning or end of the list. This might completely be the intended effect, but I preferred to ask. If beginning/end is the intended effect, maybe it could be diagonal arrows or something similar to be closer to what we have on keyboards.

**Comment by fs-sifat** [comment]
Yes, I also think the arrow keys should only nudge the list  instead of scrolling all the way to the top or bottom. 

About the scroll bar(the small letter list) can you share a screen recording. The expected behavior is as you said, quick access to the application list at those letters.

**Comment by fs-sifat** [comment]
This is the phrase you don't want to hear in an issue: "It works for me". Sorry couldn't help

**Comment by fs-sifat** [comment]
Cheers mate!
About the second question, yes that's a problem. And it looks like an easy fix

Make sure to rename the issue.
